packages/template-retail-react-app/app/components/product-tile/index.jsx

```diff
@@ -86,61 +51,18 @@ const ProductTile = (props) => {
             {...rest}
         >
             <Box {...styles.imageWrapper}>
```

Comment:
ProductTile has had its styles extracted out and saved in Chakra theme. But only several components so far have extracted the styles in this manner. Most components have inline styles (using Chakra's style props).

For better extensibility, we need to be more consistent and start extracting out the inline styles.